crates/fluvio-test/src/tests/consumer.rs

```diff
@@ -233,8 +233,7 @@ async fn get_multi_stream(
     )
 }
 
-// Default to using the producer test's topic
-#[fluvio_test(name = "consumer", topic = "producer-test")]
+#[fluvio_test(name = "consumer", topic = "consumer-test")]
```

Comment:
This still needs to be addressed for shipping.

Reply:
I pushed a fix for the consumer test, but I think there is a might be a overall bug in the test-driver: https://github.com/infinyon/fluvio/issues/2695

Reply:
k